tutorials/v7/line.cxx

```diff
@@ -30,9 +30,41 @@ void line()
 
    // Create a canvas to be displayed.
    auto canvas = Experimental::TCanvas::Create("Canvas Title");
-   auto line       = std::make_shared<Experimental::TLine>(0. ,0. ,1. , 1.);
-   canvas->Draw(line);
-      
+ 
+
+
+   for (double i = 0; i < 360; i+=1){
+        double ang = i * TMath::Pi() / 180;
+        double x = 0.3*TMath::Cos(ang) + 0.5;
+        double y = 0.3*TMath::Sin(ang) + 0.5;
+
+        auto line = std::make_shared<Experimental::TLine>(0.5 ,0.5 ,x, y);
+
+        auto col = Experimental::TColor(0.0025*i, 0, 0);
+        line->GetOptions().SetLineColor(col);
+        line->GetOptions().SetLineWidth(1);
+        //line->GetOptions().SetLineColorAlpha(0.45);
+        canvas->Draw(line);
+
+    }
+
+    auto line  = std::make_shared<Experimental::TLine>(0. ,0. ,1. ,1.);
+    auto line1 = std::make_shared<Experimental::TLine>(0.1 ,0.1 ,0.9 ,0.1);
+    auto line2 = std::make_shared<Experimental::TLine>(0.9 ,0.1 ,0.9 ,0.9);
+    auto line3 = std::make_shared<Experimental::TLine>(0.9 ,0.9 ,0.1 ,0.9);
+    auto line4 = std::make_shared<Experimental::TLine>(0.1 ,0.1 ,0.1 ,0.9);
+    auto line0 = std::make_shared<Experimental::TLine>(0. ,1. ,1. ,0.);
+
+
+    canvas->Draw(line);
```

Comment:
You should be able to `Draw(Experimental::TLine(0. ,0. ,1. ,1.))` etc. Does that work? If not I'll fix it :-)